src/Acuminator/Acuminator.Vsix/Resources/VSIXResource.resx

```diff
@@ -250,16 +250,16 @@
     <value>Enable additional diagnostics for ISV Solution Certification</value>
   </data>
   <data name="Setting_CodeAnalysis_StaticAnalysisEnabled_Description" xml:space="preserve">
-    <value>Enable Acuminator static code analysis for Visual Studio solution.</value>
+    <value>Enable Acuminator static code analysis for Visual Studio solutions.</value>
```

Comment:
Were these strings reviewed?

Reply:
@metallmonster  yes by @k-s-popova. See the related jira item